No go.mod file so go install does not work

# Summary
This repo lacks a go.mod file which is needed in order to specify that the minimum version of go required is 1.18
# Steps to Reproduce
1. run `go install github.com/patrickbr/gtfstidy@latest`

Then I get this error message:
```
# github.com/patrickbr/gtfstidy/processors
embedding interface element ~[]*gtfs.Trip requires go1.18 or later (-lang was set to go1.16; check go.mod)
../../go/pkg/mod/github.com/patrickbr/gtfstidy@v0.0.0-20240229214308-890a1feed801/processors/servicenonoverlapper.go:70:8: implicit function instantiation requires go1.18 or later (-lang was set to go1.16; check go.mod)
```
# References
https://go.dev/ref/mod#go-mod-file-go

--- a/go.mod
+++ b/go.mod
@@ -1,0 +1,16 @@
+module github.com/patrickbr/gtfstidy
+
+go 1.18
+
+require (
+	github.com/patrickbr/gtfsparser v0.0.0-20240120174022-f85e0e2e3ab2
+	github.com/patrickbr/gtfswriter v0.0.0-20231029093346-d2fe6aa60cc5
+	github.com/paulmach/go.geojson v1.5.0
+	github.com/spf13/pflag v1.0.5
+	golang.org/x/exp v0.0.0-20240314144324-c7f7c6466f7f
+)
+
+require (
+	github.com/klauspost/compress v1.17.7 // indirect
+	github.com/valyala/fastjson v1.6.4 // indirect
+)

